mqttclients/c/mosquitto_client_extensions/mqtt_setup.c

```diff
@@ -180,16 +180,20 @@ bool mqtt_client_set_connection_settings(mqtt_client_connection_settings* connec
   set_successfully = set_successfully
       && set_char_connection_setting(&connection_settings->hostname, "MQTT_HOST_NAME", true);
   set_successfully = set_successfully
-      && set_int_connection_setting(&connection_settings->tcp_port, "MQTT_TCP_PORT", 8883);
-  set_successfully = set_successfully
-      && set_bool_connection_setting(&connection_settings->use_TLS, "MQTT_USE_TLS", true);
+      && set_int_connection_setting(
+                         &connection_settings->tcp_port, "MQTT_TCP_PORT", DEFAULT_TCP_PORT);
   set_successfully = set_successfully
       && set_bool_connection_setting(
-                         &connection_settings->clean_session, "MQTT_CLEAN_SESSION", true);
+                         &connection_settings->use_TLS, "MQTT_USE_TLS", DEFAULT_USE_TLS);
   set_successfully
       = set_successfully
+      && set_bool_connection_setting(
```

Comment:
Not a big fan of this style of conditional: it's hard to read and understand that these are not going to execute if set successfully is false.
Instead, I would rather have a macro (if_failed_return) or if/else statements that are making things clear (`if (set successfully) {...}.`) with the same effect but easy to read.